src/spot_moveit_config/config/spot.srdf

```diff
@@ -26,132 +24,69 @@
         <joint name="rear_right_hip_y"/>
         <joint name="rear_right_knee"/>
     </group>
-    <group name="gripper">
-        <joint name="arm_gripper"/>
+    <group name="arm">
+        <chain base_link="body" tip_link="arm_link_wr1"/>
+    </group>
+    <group name="end_effector">
+        <joint name="arm_f1x"/>
     </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="default_standing" group="spot_body">
+    <group_state name="standing" group="body">
         <joint name="front_left_hip_x" value="0"/>
-        <joint name="front_left_hip_y" value="0.872665"/>
-        <joint name="front_left_knee" value="-1.44862"/>
-
+        <joint name="front_left_hip_y" value="0"/>
+        <joint name="front_left_knee" value="-1.52385"/>
         <joint name="front_right_hip_x" value="0"/>
-        <joint name="front_right_hip_y" value="0.872665"/>
-        <joint name="front_right_knee" value="-1.44862"/>
-
+        <joint name="front_right_hip_y" value="0"/>
+        <joint name="front_right_knee" value="-1.52023"/>
         <joint name="rear_left_hip_x" value="0"/>
-        <joint name="rear_left_hip_y" value="0.872665"/>
-        <joint name="rear_left_knee" value="-1.44862"/>
-
+        <joint name="rear_left_hip_y" value="0"/>
+        <joint name="rear_left_knee" value="-1.53003"/>
         <joint name="rear_right_hip_x" value="0"/>
-        <joint name="rear_right_hip_y" value="0.872665"/>
-        <joint name="rear_right_knee" value="-1.44862"/>
+        <joint name="rear_right_hip_y" value="0"/>
+        <joint name="rear_right_knee" value="-1.52531"/>
     </group_state>
```

Comment:
Review joint limits in the `standing` state.

Reply:
The current state (HX = 0°, HY = 0°, KN variations near -87.3° or -87.6°) results in Spot's legs pointing straight down and then bending ~90° forward. We instead want Spot's HY to be 50° (angled back at the hip) and KN to be -83° (maybe?) to create a default standing pose.

Reply:
Now fixed.